cocos/renderer/CCTexture2D.cpp

```diff
@@ -1444,4 +1464,9 @@ void Texture2D::removeSpriteFrameCapInset(SpriteFrame* spriteFrame)
     }
 }
 
+void Texture2D::setAutoGenerateMipmap(bool generateMipmaps)
+{
+    _generateMipmaps = generateMipmaps;
```

Comment:
It no break compatibility, user code call generateMipmap after initWithMipmaps, and it works well as it is now.
Build log message DEPRECATED.